pkg/api/v1/testkube/model_test_workflow_execution.go

```diff
@@ -49,6 +49,7 @@ type TestWorkflowExecution struct {
 	// whether webhooks on the execution of this test workflow are disabled
 	DisableWebhooks bool                                        `json:"disableWebhooks,omitempty"`
 	Tags            map[string]string                           `json:"tags,omitempty"`
+	Target          *ExecutionTarget                            `json:"target,omitempty"`
```

Comment:
As above, it is not needed. There is already `RunnerTarget` and `RunnerOriginalTarget`